WORKSPACE

```diff
@@ -391,6 +394,11 @@ hazel_repositories(
         , ("shake", {"version": "0.17.8", "sha256": "ade4162f7540f044f0446981120800076712d1f98d30c5b5344c0f7828ec49a2"})
         , ("filepattern", {"version": "0.1.1", "sha256": "f7fc5bdcfef0d43a793a3c64e7c0fd3b1d35eea97a37f0e69d6612ab255c9b4b"})
         , ("terminal-progress-bar", {"version": "0.4.0.1", "sha256": "c5a9720fcbcd9d83f9551e431ee3975c61d7da6432aa687aef0c0e04e59ae277"})
+        # Deps for haskell LSP
+        , ("rope-utf16-splay" , {"version": "0.2.0.0", "sha256": "83d1961bf55355da49a6b55d6f58d02483eff1f8e6df53f4dccdab1ac49e101d"})
+        # This is a special version of Haskell LSP without GPL dependencies
+        , ("haskell-lsp-types", {"url": "https://github.com/DavidM-D/haskell-lsp/archive/{}.zip".format(HASKELL_LSP_COMMIT), "sha256": "{}".format(HASKELL_LSP_HASH), "stripPrefix": "haskell-lsp-{}/haskell-lsp-types".format(HASKELL_LSP_COMMIT)})
```

Comment:
Before finishing we want to fork these to digital-asset. Note that I've changed the bazel syntax here to make this easier, but it will conflict.

Reply:
Will do, I'm not sure what the overhead of that is, but I'll do it when I'm looking to merge as it's not really a blocker

Reply:
Yep, more just a heads up.